app/scripts/modules/core/src/core.module.ts

```diff
@@ -0,0 +1,118 @@
+import { module } from 'angular';
+
+import 'bootstrap/dist/css/bootstrap.css';
+import 'jquery-ui';
+// Must come after jquery-ui - we want the bootstrap tooltip, JavaScript is fun
+import 'bootstrap/dist/js/bootstrap.js';
+
+import 'font-awesome/css/font-awesome.css';
+import 'react-select/dist/react-select.css';
+import 'ui-select/dist/select.css';
+
+import UI_ROUTER from '@uirouter/angularjs';
+require('@uirouter/angularjs/release/stateEvents');
```

Comment:
@anotherchrisberry I deleted the copy of the shim from spinnaker and imported the one from the npm package here